src/Entries/Entry.php

```diff
@@ -734,6 +734,10 @@ public function resolveGqlValue($field)
             return $this->site();
         }
 
+        if ($field === 'parent') {
+            return optional($this->augmentedValue('parent'))->entry();
```

Comment:
Was there a specific reason you used `$this->augmentedValue('parent')` instead of just `$this->parent()`?
If not, I'd do that.

Reply:
Definitely had a reason for that... 🧐 I'll have to check if it is still necessary.

Reply:
Well, $this->parent() is working just fine in my manual tests and in the unit tests now, so there you go!